src/components/cards/DPICard.tsx

```diff
@@ -1,123 +0,0 @@
-import * as i18n from '@/languages';
-import React, { useCallback } from 'react';
-import LinearGradient from 'react-native-linear-gradient';
-import { useNavigation } from '../../navigation/Navigation';
-import { ButtonPressAnimation } from '../animations';
-import { CoinIcon } from '../coin-icon';
-import { analyticsV2 } from '@/analytics';
-import {
-  Box,
-  ColorModeProvider,
-  Column,
-  Columns,
-  globalColors,
-  Stack,
-  Text,
-} from '@/design-system';
-import { useAccountSettings } from '@/hooks';
-import store from '@/redux/store';
-import { DPI_ADDRESS } from '@/references';
-import Routes from '@/navigation/routesNames';
-import { ethereumUtils } from '@/utils';
-import { GenericCard, Gradient } from './GenericCard';
-import { ORB_SIZE } from './reusables/IconOrb';
-import { useRoute } from '@react-navigation/native';
-
-const TRANSLATIONS = i18n.l.cards.dpi;
-const CARD_GRADIENT: Gradient = {
-  colors: ['#6D58F5', '#A970FF'],
-  start: { x: 0, y: 0 },
-  end: { x: 1, y: 0 },
-};
-const BUTTON_GRADIENT: Gradient = {
-  colors: ['#5236C2', '#7533D6'],
-  start: { x: 0, y: 0 },
-  end: { x: 1, y: 0 },
-};
-
-export const DPICard = () => {
-  const { nativeCurrency } = useAccountSettings();
-  const { navigate } = useNavigation();
-  const { name: routeName } = useRoute();
-  const cardType = 'stretch';
-
-  const handlePress = useCallback(() => {
-    const asset = ethereumUtils.formatGenericAsset(
-      store.getState().data?.genericAssets?.[DPI_ADDRESS],
-      nativeCurrency
-    );
-
-    analyticsV2.track(analyticsV2.event.cardPressed, {
-      cardName: 'DPICard',
-      routeName,
-      cardType,
-    });
-
-    navigate(Routes.TOKEN_INDEX_SHEET, {
-      asset,
-      backgroundOpacity: 1,
-      cornerRadius: 39,
-      fromDiscover: true,
-      type: 'token_index',
-    });
-  }, [nativeCurrency, navigate, routeName]);
-
-  return (
-    <ColorModeProvider value="darkTinted">
-      <GenericCard
-        gradient={CARD_GRADIENT}
-        color="#8D65FA"
-        onPress={handlePress}
-        testID="dpi-button"
```

Comment:
could remove e2e test for this, it's skipped for now but could save trouble later